meta.go

```diff
@@ -74,3 +74,24 @@ func (m Meta) Clone() Meta {
 	m.Encoder = m.Encoder.Clone()
 	return m
 }
+
+// Enabled returns true if logging a message at a particular level is enabled.
+func (m Meta) Enabled(lvl Level, msg string) bool {
```

Comment:
Good qn, I think having the msg only encourages sampling-like things; will drop.